lib/services/android-project-service.ts

```diff
@@ -517,12 +517,18 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		if (this.$androidToolsInfo.getToolsInfo().androidHomeEnvVar) {
 			const gradlew = this.$hostInfo.isWindows ? "gradlew.bat" : "./gradlew";
 
+			const localArgs = [...gradleArgs];
+			if (this.$logger.getLevel() === "INFO") {
+				localArgs.push("--quiet");
+				this.$logger.info(`${gradlew} ${localArgs.join(" ")}`);
```

Comment:
Otherwise nothing is printed. I wanted to print something that indicates that a native build had started. I am ok to go for just "Xcode build..." and "Gradle build...", should I change these?